docker-compose/docker-compose.yml

```diff
@@ -31,7 +31,7 @@ services:
     environment:
       - POSTGRES_PASSWORD=admin
     volumes:
-      - postgres-volume:/var/lib/postgresql/data
+      - postgres-volume:/var/lib/postgresql
```

Comment:
Recently there was a major postgres update. In the logs I saw that it is recommended to mount the whole postgresql folder. By doing that, it is possible to migrate data from old postgres versions to the new one. So I did that.